src/models/StyleSheet.js

```diff
@@ -197,26 +203,30 @@ class StyleSheet {
     return (this.tagMap[id] = tag)
   }
 
-  /* optimal caching: hash is known and name can be returned */
-  getNameForHash(hash: string) {
-    return this.hashes[hash]
+  /* mainly for injectGlobal to check for its id */
+  hasId(id: string) {
+    return this.tagMap[id] !== undefined
```

Comment:
Should we switch to an es6 Map now? Might be faster.

Reply:
Not enough browser support, it barely works in IE11 and we support IE9 and up: http://kangax.github.io/compat-table/es6/#test-Map :cry:

Reply:
Oh, we shouldn't be using Array.from then.